src/app/course/course-item/course-item.component.spec.ts

```diff
@@ -2,14 +2,17 @@ import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 
 import { CourseItemComponent } from "./course-item.component";
 import { ActionTypes } from "../_models/actionTypes";
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+import { DurationPipe } from "../duration.pipe";
 
```

Comment:
Talking about the imports, I would advice to organize them next way:
first group is external imports. angular, rxjs, lodash and so on
second one is external from your libraries (you have no in the project, so skip the group)
the third one should be for internal imports.

Between each group you can leave an empty line.
Also, try to use imports from index